### ISSUE TITLE
Supporting c++ highlight in WSL environment?

### ISSUE THREAD

**Issue by JackTATStraw** [jeff-hykin/cpp-textmate-grammar#400]
Hi, I found out that the better C++ syntax doesn't work when editting c++ file in windows 10 wsl mode. I wonder whether this is a bug or WSL is not supported for current version.

**Comment by matter123** [comment]
The environment should have no bearing on syntax highlighting from this extension. Do you have `C_Cpp.enhanced_colorization` enabled?

**Comment by jeff-hykin** [comment]
I'm aware of WSL, but I don't know of a WSL mode. Could you post screenshots @JackTATStraw ?

**Comment by JackTATStraw** [comment]
> e

yeah, I tested on different conditions. When the vs code is started from windows 10, it worked fine for me. However, when I start the vs code in wsl, it just doesn't work dispite the enhanced_colorization is already disabled.

**Comment by JackTATStraw** [comment]
> The environment should have no bearing on syntax highlighting from this extension. Do you have `C_Cpp.enhanced_colorization` enabled?

yeah, I tested on different conditions. When the vs code is started from windows 10, it worked fine for me. However, when I start the vs code in wsl, it just doesn't work dispite the enhanced_colorization is already disabled.

**Comment by JackTATStraw** [comment]
> I'm aware of WSL, but I don't know of a WSL mode. Could you post screenshots @JackTATStraw ?

You can find a reference here by microsoft explaining how vs code works in WSL mode: [https://code.visualstudio.com/docs/remote/wsl](url)

I do think the extension need to be caliberated for its proper execution on WSL, and here is a reference on how to do it:[https://code.visualstudio.com/api/advanced-topics/remote-extensions](url)

**Comment by jeff-hykin** [comment]
If you could show visually what is happening we can probably help figure out the problem better. In theory WSL/Windows 10/Linux/Mac shouldn't affect this at all

**Comment by JackTATStraw** [comment]
> If you could show visually what is happening we can probably help figure out the problem better. In theory WSL/Windows 10/Linux/Mac shouldn't affect this at all.
> 
> the link about WSL was helpful though

![image](https://user-images.githubusercontent.com/10207226/66921486-e7020100-f057-11e9-9371-1778649f900b.png)

here is what happend when I open it in wsl mode.
Nothing is highlighted

**Comment by jeff-hykin** [comment]
Actually that is exactly how the highlighting appears for me with the extension enabled.
![Screen Shot 2019-10-16 at 8 21 59 AM](https://user-images.githubusercontent.com/17692058/66923279-1f6cf500-efee-11e9-864b-07dd428ae772.png)

However, that could just be coincidence. This is how imports look for me:
![Screen Shot 2019-10-16 at 8 33 07 AM](https://user-images.githubusercontent.com/17692058/66924080-9951ae00-efef-11e9-892a-55c59697def9.png)

If they look that way for you, then everything is working
